src/routes/tutorial/[slug]/codemirror.css

```diff
@@ -53,61 +53,82 @@
 }
 
 .cm-editor .cm-selectionMatch {
-	background: rgba(0,0,0,0.2);
+	background: var(--selection-color);
 	color: var(--sk-text-2);
 }
 
+.cm-editor.cm-focused .cm-selectionBackground,
+.cm-editor .cm-selectionBackground,
+.cm-editor .cm-content ::selection {
+	background: var(--sk-theme-3);
+	opacity: .3;
+}
+
 .cm-editor .cm-tooltip {
+	background: var(--sk-back-2);
+	color: var(--sk-text-1);
 	border: none;
 	border-radius: 2px;
 	overflow: hidden;
 	margin: 0.4rem 0;
 	filter: drop-shadow(1px 2px 5px rgba(0,0,0,0.1));
 }
-.cm-editor .cm-tooltip-hover {}
+
+.cm-editor .cm-tooltip-hover {
+	border: none;
+}
+
```

Comment:
This fixes #360.

<img width="742" alt="image" src="https://user-images.githubusercontent.com/29677552/232085474-92f2eec6-fda0-4c87-a220-7078e737e501.png">


---

<img width="720" alt="image" src="https://user-images.githubusercontent.com/29677552/232084412-c9f7c803-d114-4780-8b01-9ab3959f396a.png">